packages/nuxi/src/commands/analyze.ts

```diff
@@ -11,62 +11,99 @@ import { defineNuxtCommand } from './index'
 export default defineNuxtCommand({
   meta: {
     name: 'analyze',
-    usage: 'npx nuxi analyze [--log-level] [rootDir]',
+    usage: 'npx nuxi analyze [--log-level] [--name] [--no-serve] [rootDir]',
```

Comment:
would it make sense to accept these as programmatic options rather than CLI options (for `analyzeCommand.invoke(args, options)`)? We do that for `dev` and could also do that here. Not sure if that would work for devtools. If so, it might avoid confusion when seeing help printed for the analyze command. (But if you think either option might be used by end users, then we can of course keep it here...)

https://github.com/nuxt/nuxt/blob/4c200e4a2a7c3472153b6fac3e81d4e911c9a50a/packages/nuxi/src/commands/dev.ts#L24

Reply:
That would also work for me. The problem is that we need to call it from a subprocess since we are starting a new build. But it could be done from DevTools by providing a CLI wrapper to invoke it - or maybe that would be better for devtools to iterate.

Reply:
What do you think would be better?

Reply:
I would still need this PR in order to support DevTools usage. Meanwhile, I will try to move some logic to DevTools.

Reply:
Oh, absolutely. I'm just wondering whether we keep these as CLI arguments or programmatic options.